resources/newrelic.ini

```diff
@@ -56,7 +52,7 @@ monitor_mode = true
 # write out a log file, it is also possible to say "stderr" and
 # output to standard error output. This would normally result in
 # output appearing in your web server log.
-log_file = ./newrelic-agent.log
+#log_file = /tmp/newrelic-agent.log
```

Comment:
This appears to be commented out, but you might want to actually configure this. A buildpack should make sure that anything writing logs writes those logs to `/tmp`. That's really the only directory that can be guaranteed will have write access at runtime (some environments will run as non-privileged users with no write access aside from `/tmp`).

Reply:
Done